src/libopensc/libopensc.exports

```diff
@@ -133,11 +135,13 @@ sc_make_cache_dir
 sc_mem_alloc_secure
 sc_mem_clear
 sc_mem_reverse
+_sc_match_atr_block
 sc_match_atr_block
 sc_path_print
 sc_path_set
 sc_pin_cmd
 sc_pkcs1_encode
+sc_pkcs1_strip_02_padding
```

Comment:
The VC build file links statically against libopensc, which could be the reason why the "correct" exports are not needed. Preferably we should use the same setup here.

With http://stackoverflow.com/questions/10842949/force-linking-a-static-library-into-a-shared-one-with-libtool the solution should look something like:
```
noinst_LTLIBRARIES = libopensc_impl.la

libopensc_impl_la_LIBADD = $(top_builddir)/src/libopensc/libopensc.la

opensc_minidriver@LIBRARY_BITNESS@_la_LIBADD = libopensc_impl.la \
...
```
Also, OpenSSL should be linked statically (i.e. requires a change to `configure.ac`)